src/utils/dictionaries/redirects.json

```diff
@@ -0,0 +1 @@
","/es/course/bootcamp-desarrollo-web-full-stack-part-time","/es/course/introduccion-programacion","/introduccion-programacion-codigo","/program","/the-program","/us/part-time","/us/course/part-time","/us/full-stack","/landing/coding-classes-part-time","/course/web-development-parttime","/coding-introduction-programming","/us/course/coding-introduction","/course/coding-introduction","/us/coding-bootcamps/full-stack-web-development-bootcamp-part-time","/course/full-stack-web-development-bootcamp-part-time","/us/course/full-stack-web-development-bootcamp-part-time","/es/machine-learning","/us/machine-learning","/ingenieria-software","/es/software-engineering","/es/course/ingenieria-software","/software-engineering","/us/software-engineering","/us/course/software-engineering-coding-bootcamp","/course/software-engineering-coding-bootcamp","/us/coding-bootcamps/software-engineering-coding-bootcamp","/coding-bootcamps/software-engineer-bootcamp","/landing/becas-partner","/landing/impulsa-tu-carrera","/landing/impulsa-tu-carrera-carne-joven","/landing/coding-classes","/landing/coding-classes-parttime","/landing/coding-course","/landing/coding-school","/landing/full-stack-programs","/landing/income-sharing-agreement","/landing/learn-to-code","/landing/programming-bootcamp","/landing/programming-courses","/landing/aprende-a-programar-santiago","/landing/bootcamp-de-programacion-santiago","/landing/clases-de-programacion-santiago","/landing/curso-de-programacion-santiago","/career","senior-mentor","/us/colombia","/colombia","/es/coding-campus/bootcamp-programacion-argentina","/us/argentina","/argentina","/us/coding-campus/coding-bootcamp-argentina","/venezuela","/caracas","/es/location/impact-hub-caracas","/es/location/caracas-venezuela","/impact-hub","/location/impact-hub","/us/location/caracas-venezuela-en","/es/location/costa-rica-san-jose","/us/location/costa-rica","/location/costa-rica","/es/location/downtown-miami-usa","/locations","/miami","/us/location/downtown-miami","/location/downtown-miami","/us/coding-campus/downtown-miami","/coding-campus/coding-bootcamp-miami","/us/coding-bootcamp-miami","/es/coding-campus/bootcamp-programacion-bolivia","/us/bolivia","/bolivia","madrid","/es/location/madrid-spain/","/es/location/madrid-espana","/location/madrid","/us/location/madrid-spain","/es/location/maracaibo-venezuela","/us/location/maracaibo","/location/maracaibo","/coding-campus/bootcamp-programacion-mexico","/us/mexico","/mexico","/coding-campus/bootcamp-programacion-uruguay","/us/uruguay","/uruguay","/location/remoto","/remoto","/es/location/enlinea-remoto","/location/online","/location/remote","/online","/us/location/online-remote","/coding-campus/bootcamp-programacion-panama","/us/panama","/panama","/coding-campus/bootcamp-programacion-ecuador","/us/ecuador","/ecuador","/chile","/es/inicio/santiago-de-chile","/es/location/santiago-chile-es","/location/santiago","/location/santiago-de-chile","/us/location/santiago-chile","/location/santiago-chile"]
```

Comment:
I have created a dictionary that obtains all the redirects in case of finding in the blogs links that point to a redirect